app/helpers/searches_helper.rb

```diff
@@ -0,0 +1,2 @@
+module SearchesHelper
```

Comment:
I think we'd prefer not to include the stub files unless we have a specific need.